seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-sftp-e2e/src/test/resources/text/multiple_fake_to_sftp_file_text.conf

```diff
@@ -0,0 +1,103 @@
+#
```

Comment:
Please rename the test case file name. I can not get the difference of two config from config name.

Reply:
updated